Cookbook/Technical-Documents/WhatIDontKnow.md

```diff
@@ -0,0 +1,183 @@
+
+## In the tower of Babel
+When working in such a vibrant and dynamic environment it is very easy to lose a track of things that should happen. This is a shortlist of topics that should help you to get through the onboarding time.
+
+### Communication and Collaboration
+Effective and open communication is crucial in a company like Babylon. Without it, everything falls apart. If there is one thing you should remember from this guide let me suggest this:
+Context is king!
+
+When you’re communicating digitally, you never quite know what the other person is doing at that moment. They might be at their desk just like you are, or they may be frantically rushing to a meeting, only responding “Yes” to your question and not elaborating because they don’t have time.
+
+Without understanding the other person’s context, you might think that person doesn’t care about the issue you brought up when they’re really just running to a meeting room. With the information on that person’s context, all of a sudden, the curt answers make sense: It’s not that your coworker doesn’t care, they are just indisposed at the moment.
+
+Prefacing communication with your context can help to prevent any miscommunication when things are out of the ordinary. Let team members know when you are heads down on a project and can’t respond to questions right away.
+
+Over communicating is always better than making assumptions.
+
+To help you with a difficult art of digital communication, here are some resources for you:
+
+[Remote team communication](https://zapier.com/learn/remote-work/remote-team-communication/)
+
+Why you should read it: This will give you a bunch of tips on how to communicate better digitally.
+
+[Remote communication](https://about.gitlab.com/2016/03/23/remote-communication/index.html)
+
+Why you should read it: It has some universal commandments about communicating asynchronously (between humans ;) )
+
+[Communication: how to be a better software developer](https://medium.com/techspiration-ideas-making-it-happen/communication-how-to-be-a-better-software-developer-869c50767701)
+
+Why you should read it: A bit longer read, but presents good practices we all should strive to achieve.
+
+### Am I missing something?
+This is our FAQ section. Every question that you may have have a short answer and/or links to additional resources. If during your onboarding you ever find yourself asking a question that is not on this list, create a new item, find an answer with your buddy and update the board.
+
+#### List of events you should have in your calendar
+- Your squad’s daily/retro/ planning
+- Babylon standup
+- Frontend demo
+- Proposal meetings
+- PR party
+- Lunch and Learn
+- iOS Banter
+- 1-on-1 with your manager
+
+#### List of documents you should star on the Google drive
```

Comment:
I'd remove links from this section, they change periodically and would become outdated unless updated here. Let's keep the title generic and no links